include/ocpp/v201/device_model_storage_sqlite.hpp

```diff
@@ -40,6 +40,13 @@ class DeviceModelStorageSqlite : public DeviceModelStorage {
     bool set_variable_attribute_value(const Component& component_id, const Variable& variable_id,
                                       const AttributeEnum& attribute_enum, const std::string& value) final;
 
+    bool set_monitoring_data(const SetMonitoringData& data) final;
```

Comment:
It looks like its more efficient to use a std::map here:
https://github.com/EVerest/libocpp/pull/653/files#diff-646ba26ec38020a5c0b7ae2132ec4c2719ffb6efd38243bb37ac140aaca096ecR21

`std::map<int, VariableMonitoring> monitors;`

The key is the id of the monitor

This function should therefore return the id of the monitor after it has been added/updated in the storage. The id can be used as the key to the monitors map of `VariableMetaData`